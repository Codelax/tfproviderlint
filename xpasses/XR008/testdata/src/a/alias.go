package a

import (
	"context"
	e "os/exec"
)

func fAlias() {
	failingAlias := e.CommandContext // want "avoid os/exec.CommandContext"

	e.CommandContext(context.Background(), "true") // want "avoid os/exec.CommandContext"

	failingAlias(context.Background(), "true")
}
