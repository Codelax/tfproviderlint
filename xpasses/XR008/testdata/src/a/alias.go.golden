package a

import (
	"context"
	e "os/exec"
)

func fAlias() {
	failingAlias := // want "avoid os/exec.CommandContext"

		// want "avoid os/exec.CommandContext"

		failingAlias(context.Background(), "true")
}
